scss/_settings_colors.scss

```diff
@@ -78,12 +78,12 @@ $colors--light-theme--text-muted: #666 !default;
 
 $colors--light-theme--background-default: #fff !default;
 $colors--light-theme--background-alt: #f7f7f7 !default;
-$colors--light-theme--background-highlight: #f7f7f7 !default;
+$colors--light-theme--background-highlight: rgba($color-x-dark, 0.03) !default;
 $colors--light-theme--background-overlay: transparentize($color-dark, 0.15) !default;
 
-$colors--light-theme--border-default: #cdcdcd !default;
-$colors--light-theme--border-high-contrast: #717171 !default;
-$colors--light-theme--border-low-contrast: #e5e5e5 !default;
+$colors--light-theme--border-default: rgba($color-x-dark, 0.2) !default;
+$colors--light-theme--border-high-contrast: rgba($color-x-dark, 0.55) !default;
```

Comment:
It is in between, but to be on the safe side of AA criteria I would round to 0.56